charts/workflows-cluster/templates/ingress-tls-secret.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.ingress.enabled }}
```

Comment:
Dunno how I managed to not add this condition despite having the option in the `values.yaml`